Makefile

```diff
@@ -13,14 +13,14 @@ load-design-system-templates:
 	pipenv run ./scripts/load_templates.sh $(DESIGN_SYSTEM_VERSION)
 
 lint:
-	pipenv check ./response_operations_ui ./tests -i 51499
+	pipenv check ./response_operations_ui ./tests -i 51499 -i 51668
```

Comment:
51668? I am only getting one now, the new one for certifi which can be fixed with a bump, see frontstage. It might be your version of pipenv